spec.html

```diff
@@ -8794,6 +8834,24 @@ <h1>CreateBuiltinFunction ( _steps_, _internalSlotsList_ [ , _realm_ [ , _protot
       </emu-alg>
       <p>Each built-in function defined in this specification is created by calling the CreateBuiltinFunction abstract operation.</p>
     </emu-clause>
+
+    <emu-clause id="sec-createbuiltingeneratorfunction" aoid="CreateBuiltinGeneratorFunction">
+      <h1>CreateBuiltinGeneratorFunction ( _steps_ [ , _iteratorKind_ [ , _generatorPrototype_ [ , _generatorInternalSlotsList_ ]]]) </h1>
+      <p>The abstract operation CreateBuiltinGeneratorFunction create a <dfn id="sec-builtin-generator-function">built-in generator function</dfn> object. The generator object it returns can have extra internal slots.</p>
+      <p>The abstract operation CreateBuiltinGeneratorFunction takes arguments _steps_, and optional arguments _generatorInternalSlotsList_ (a List of internal slot names that the returned generator has), _generatorPrototype_ (the prototype of the returned generator, should inherit from %IteratorPrototype%). It performs the following steps when called:</p>
+      <emu-alg>
+        1. Assert: _steps_ is either a set of algorithm steps or other definition of a function's behaviour provided in this specification. It can contain Yield abstract operations to yield an IteratorResult object.
+        1. If _generatorPrototype_ is not present, set _generatorPrototype_ to %Iterator.prototype%.
+        1. Append &laquo; [[GeneratorState]], [[GeneratorContext]], [[IteratorKind]] &raquo; to _generatorInternalSlotsList_.
```

Comment:
this reads like it's doing 
```js
_generatorInternalSlotsList_.push(['[[GeneratorState]]', '[[GeneratorContext]]', '[[IteratorKind]]'])
```
when i think you want 
```js
_generatorInternalSlotsList_.push(...['[[GeneratorState]]', '[[GeneratorContext]]', '[[IteratorKind]]'])
```
i'm not sure how to word this clearer but it needs something :-)